pagopa/pn-frontend
# PN-2272 fix: enables attachments elimination in NewNotification page

## Short description
Changes forEach to map in deleteDocumentHandler function in packages/pn-pa-webapp/src/pages/components/NewNotification/Attachments.tsx

## List of changes proposed in this pull request
- Changes forEach to map in deleteDocumentHandler function

## How to test
- Startpn-pa-webapp
- Create a new notification 
- Reach the payment models section 
- return to attachments section
- add and delete attachments
- the attachments should be deleted correctly
